parser/src/command/assign.rs

```diff
@@ -68,7 +68,7 @@ impl AssignCommand {
             } else {
                 return Err(toks.error(ParseError::NoUser));
             }
-        } else if let Some(Token::Word("release-assignment")) = toks.peek_token()? {
+        } else if let Some(Token::Word("release-assignment" | "unclaim")) = toks.peek_token()? {
```

Comment:
Could you also update the documentation on the enum above.

Reply:
We'll also need to update the user-documentation on the forge at https://github.com/rust-lang/rust-forge/blob/master/src/triagebot/issue-assignment.md

Reply:
I plan on updating the forge once this is merged.